docs/source/posts/2020/2020-07-27-week-9-lenix.rst

```diff
@@ -0,0 +1,37 @@
+Merging SDF primitives.
+===========================================
+
+.. post:: July 27 2020
+   :author: Lenix Lobo
+   :tags: google
+   :category: gsoc
+
+Make sure to check out Project FURY : <https://github.com/fury-gl/fury>
+
+Hey Everyone! 
+This week, Merging SDF primitives.
+
+What did you do this week?
+--------------------------
+Since GSoC started I have been working on adding support for raymarching based SDF actors as primitives in the FURY codebase. This week with the release of FURY 0.6.0 , the task assigned to me was to complete the remaining parts of the SDF actor including tests and tutorial. THe SDF actor is now part of the FURY actor and can be accessed using sdf_actor.
+Currently we support , ellipsoids, spheres and torus as primitive options. As expected, SDF based actors have shown tremendous performance improvements over traditional polygon based actor.
+
+Despite using 100,000 torus the FPS is higher than 60 :
+
+.. image:: https://raw.githubusercontent.com/lenixlobo/fury-outputs/master/blog-week-9.gif
+
+10,000 actors :
+
+.. image:: https://raw.githubusercontent.com/lenixlobo/fury-outputs/master/blog-week-9b.gif
+
+I also made a tutorial for new users to get started with 
+
+<https://fury.gl/latest/auto_tutorials/04_shaders/viz_sdfactor.html#sphx-glr-auto-tutorials-04-shaders-viz-sdfactor-py>
```

Comment:
syntax error, link to improve

Reply:
WIll make the changes @skoudoro 
 But don't you think it would be better to merge the blogs when the entire GSoC period is done, Instead of merging some now and some later?

Reply:
I was thinking the same but finally, it make more sense to do do it by evaluation period. Here, it represents 2 evaluation periods. And your next PR will be merged only on September.